### ISSUE TITLE
Adding Liquid Support

### ISSUE THREAD

**Issue by fusion809** [burodepeper/language-markdown#97]
Hi,

Liquid support in this package would be a great help for the bloggers among us. This is an issue that started at the [`language-gfm`](https://github.com/atom/language-gfm/issues/145) repo. Here is some liquid content, so you can test out your tweaks:

```md
{% include Code/codeu.html line1="git clone https://aur.archlinux.org/atom-editor-bin.git" line2="cd atom-editor-bin" line3="makepkg -si --noconfirm" %}
{% endcapture %}

<!-- item2 -->
{% capture item2 %}
{% include Links/distros/centos7.html puncr="." %} Atom is available from the [Mosquito Copr](https://copr.fedorainfracloud.org/coprs/mosquito/atom/) and can be installed via running:

{% include Code/coder.html line1="yum copr enable mosquito/atom" line2="yum install -y atom" %}
{% endcapture %}
```

Thanks for your time,
Brenton

**Comment by burodepeper** [comment]
Thanks. I'll have a look at it! Is this the same `liquid` that Shopify uses in their themes btw?

Reference: https://github.com/atom/language-gfm/issues/145

**Comment by fusion809** [comment]
Sure is, but it is also the templating engine used by Jekyll and a few other static site generators. I wrote a [blog post on running Atom on Linux](https://fusion809.github.io/running-atom-on-linux) that is extensively powered by Liquid templates.

**Comment by burodepeper** [comment]
Thanks. I'll update here as soon as I have something.

**Comment by burodepeper** [comment]
@fusion809 I've implemented it in the `dev` branch. It turned out to be as easy as I hoped. There's still a little bug (see #99) that's keeping me from releasing this, but I hope to get that out of the way quite soon.

**Comment by burodepeper** [comment]
Bug squashed, published `v0.12`. Give it a try, and let me know if you run into any issues. Be wary that `language-markdown` disables `language-gfm` by default (to minimize any conflicts) in case you want to switch back.
